### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by kogdenko** [thom311/libnl#355]
```
./include/linux-private/linux/ipv6.h:134:2: error: expected specifier-qualifier-list before ‘__struct_group’

  134 |  __struct_group(/* no tag */, addrs, /* no attrs */,
```

```
uname  -a

Linux mtslaptop 5.15.0-76-generic #83~20.04.1-Ubuntu SMP Wed Jun 21 20:23:31 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

```
gcc --version

gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
```

**Comment by thom311** [comment]
need to also import `linux/stddef.h` header.

**Comment by thom311** [comment]
fixed by a1e9fb3d5d3c7184f51dce9e5f6c0977038a9354.
